packages/datadog-plugin-hapi/src/index.js

```diff
@@ -82,6 +115,39 @@ module.exports = [
       this.unwrap(Request.prototype, '_execute')
     }
   },
+  {
+    name: '@hapi/hapi',
+    versions: ['>=18.2'],
+    file: 'lib/core.js',
+    patch (Core, tracer, config) {
+      this.wrap(Core.prototype, '_dispatch', createWrapDispatch(tracer, config))
+    },
+    unpatch (Core) {
+      this.unwrap(Core.prototype, '_dispatch')
+    }
+  },
+  {
+    name: '@hapi/hapi',
+    versions: ['>=17.9 <18'],
+    file: 'lib/core.js',
+    patch (Core, tracer, config) {
+      this.wrap(Core.prototype, '_dispatch', createWrapDispatch(tracer, config))
+    },
+    unpatch (Core) {
+      this.unwrap(Core.prototype, '_dispatch')
+    }
+  },
+  {
+    name: 'hapi',
+    versions: ['18 - 18.1'],
```

Comment:
Block is not needed as there versions are guaranteed since there are no new releases.